addons/sale/models/sale.py

```diff
@@ -36,6 +36,14 @@ def _get_default_require_signature(self):
     def _get_default_require_payment(self):
         return self.env.company.portal_confirmation_pay
 
+    @api.model
+    def action_quotation_sent(self):
+        if self.filtered(lambda so: so.state != 'draft'):
+            raise Warning('Only draft orders can be marked as sent directly.')
```

Comment:
`odoo.exceptions.Warning` have long been a deprecated alias to `UserError`

Reply:
@MiquelRForgeFlow Done Thank you :+1: